src/controllers/userController.js

```diff
@@ -106,6 +106,146 @@ class UserController {
       });
     }
   }
+
+  /**
+ *
+ *@description Follow User
+ * @static
+ * @param {object} req
+ * @param {object} res
+ * @returns {object} res
+ * @memberof UserController
+ */
+  static async followUser(req, res) {
+    try {
+      const { userId } = req.params;
```

Comment:
create a middleware to validate that this user exists, instead of doing it in the controller